### PR TITLE
Docs: Flowchart - minor verbiage update

### PR DESCRIPTION
**Pull request by huynhicode**
## :bookmark_tabs: Summary

Minor update to the Flowchart doc.

Resolves #<your issue id here> n/a

## :straight_ruler: Design Decisions

Clearer verbiage for the first two paragraphs.

### :clipboard: Tasks

Make sure you

- [x] :book: have read the [contribution guidelines](https://github.com/mermaid-js/mermaid/blob/develop/CONTRIBUTING.md)
- [ ] :computer: have added unit/e2e tests (if appropriate)
- [ ] :notebook: have added documentation (if appropriate)
- [x] :bookmark: targeted `develop` branch

### CODE REVIEW

File: packages/mermaid/src/docs/syntax/flowchart.md [hunk 1]
```diff
@@ -5,11 +5,9 @@ outline: 'deep' # shows all h3 headings in outline in Vitepress
 
 # Flowcharts - Basic Syntax
 
-All Flowcharts are composed of **nodes**, the geometric shapes and **edges**, the arrows or lines. The mermaid code defines the way that these **nodes** and **edges** are made and interact.
+Flowcharts are composed of **nodes** (geometric shapes) and **edges** (arrows or lines). The Mermaid code defines how nodes and edges are made and accommodates different arrow types, multi-directional arrows, and any linking to and from subgraphs.
 
-It can also accommodate different arrow types, multi directional arrows, and linking to and from subgraphs.
-
-> **Important note**: Do not type the word "end" as a Flowchart node. Capitalize all or any one the letters to keep the flowchart from breaking, i.e, "End" or "END". Or you can apply this [workaround](https://github.com/mermaid-js/mermaid/issues/1444#issuecomment-639528897).
+> **Important**: If you are using the word "end" in a Flowchart node, capitalize the entire word or any of the letters (e.g., "End" or "END"), or apply this [workaround](https://github.com/mermaid-js/mermaid/issues/1444#issuecomment-639528897). Typing "end" in all lowercase letters will break the Flowchart.
```
Comment (sidharthv96) [inline packages/mermaid/src/docs/syntax/flowchart.md:10]:
```suggestion
```warning
If you are using the word "end" in a Flowchart node, capitalize the entire word or any of the letters (e.g., "End" or "END"), or apply this [workaround](https://github.com/mermaid-js/mermaid/issues/1444#issuecomment-639528897). Typing "end" in all lowercase letters will break the Flowchart.
\``` Remove the \ here. Suggestion didn't include the ending triplequote.
```

Comment (huynhicode) [reply packages/mermaid/src/docs/syntax/flowchart.md:10]: @sidharthv96 

The yellow text on yellow background is difficult to read:

Screenshot 2023-04-19 at 12 13 28 PM

Comment (sidharthv96) [reply packages/mermaid/src/docs/syntax/flowchart.md:10]: I agree, We should fix that with style changes in a later PR as it'll affect the whole site. 

But, if we don't use the proper warning format here, we'll forget to add it once the style is fixed.

Comment (sidharthv96) [reply packages/mermaid/src/docs/syntax/flowchart.md:10]: image

Vitepress warning looks fine. Maybe it's a different version.

Comment (sidharthv96) [reply packages/mermaid/src/docs/syntax/flowchart.md:10]: They have fixed it in the new version. I've upgraded vitepress in develop. 
image

**Comment by huynhicode** [comment]
@sid Thanks!

File: packages/mermaid/src/docs/syntax/flowchart.md [hunk 2]
```diff
@@ -5,11 +5,11 @@ outline: 'deep' # shows all h3 headings in outline in Vitepress
...
+Flowcharts are composed of **nodes** (geometric shapes) and **edges** (arrows or lines). The Mermaid code defines how nodes and edges are made and accommodates different arrow types, multi-directional arrows, and any linking to and from subgraphs.
 
-It can also accommodate different arrow types, multi directional arrows, and linking to and from subgraphs.
-
-> **Important note**: Do not type the word "end" as a Flowchart node. Capitalize all or any one the letters to keep the flowchart from breaking, i.e, "End" or "END". Or you can apply this [workaround](https://github.com/mermaid-js/mermaid/issues/1444#issuecomment-639528897).
+```warning
+If you are using the word "end" in a Flowchart node, capitalize the entire word or any of the letters (e.g., "End" or "END"), or apply this [workaround](https://github.com/mermaid-js/mermaid/issues/1444#issuecomment-639528897). Typing "end" in all lowercase letters will break the Flowchart.
+\``` Remove the \ here. Suggestion didn't include the ending triplequote.
```
Comment (sidharthv96) [inline packages/mermaid/src/docs/syntax/flowchart.md:12]: @huynhicode 😬 github UI wasn't showing the backticks. Can you fix this?

**Review by sidharthv96 — CHANGES_REQUESTED** [review]

File: packages/mermaid/src/docs/syntax/flowchart.md (hunk 2 above)
Comment (huynhicode) [reply packages/mermaid/src/docs/syntax/flowchart.md]: Gah, I can see why using GitHub UI is not as ideal as making updates in an IDE.

Looks nice now - thanks for your help @sidharthv96 !

Screenshot 2023-05-02 at 11 42 33 PM

**Comment by knsv** [comment]
@huynhicode @sidharthv96 What is the status on this PR?

**Pull request merged**